src/en/credentials.md

```diff
@@ -214,12 +226,90 @@ juju add-credential aws -f mycreds.yaml --replace
 This will overwrite existing credential information, so make sure all current
 credentials are contained in the file, not just the new or changed one.
 
+Replacing credentials in this way does not update credentials currently in use
+(on an existing controller/cloud). See next section for that. The
+`add-credential` command is always "pre-bootstrap" in nature.
+
+### Updating credentials
+
+To update credentials currently in use (i.e. cached on the controller) the
+`update-credential` command is used. The requirements for using this command,
+as compared to the initial `juju bootstrap` (or `juju add-model`) command, are:
+
+ - same cloud name
+ - same Juju username (logged in)
+ - same credential name
+
+The update is a two-step process. First change the credentials locally with the
+`add-credential` command (in conjunction with the `--replace` option) and then
+upload those credentials to the controller.
+
+Below, we log in with the correct Juju username (e.g. 'admin'), change the
+contents of the credential called 'joe', and then update them on a Google cloud
+controller:
+
+```bash
+juju add-credential --replace joe
+juju update-credential google joe
+```
+
+!!! Warning:
+    It is not possible to update the credentials if the initial credential name
+    is unknown. This restriction will be removed in an upcoming release of
+    Juju.
+
+####  Updating credentials with a different Juju user
```

Comment:
Thats' not right. This effectively is talking about "replacing' a model credential on the controller, i.e. changing 'cloud-name_userA_credential-name' with 'cloud-name_userB_credential-name'.

This is not possible. You must be the same the user.

There have been cases in the field where credential owner has left the company. These scenarios, until my rework lands, cannot be dealt with without db surgery.

Reply:
I understand, but the instructions are meant to express what a normal person wants to do. These steps *do* "update the credentials on the controller" by way of "using a different Juju user".